README.md

```diff
@@ -259,7 +294,10 @@ $ ipfs get /ipns/dist.ipfs.io/go-ipfs/$VERSION/go-ipfs_$VERSION_windows-amd64.zi
 
 ## Getting Started
 
-See also: https://docs.ipfs.io/introduction/usage/
+### Usage
+
+[![docs: Command-line quick start](https://img.shields.io/static/v1?label=docs&message=Command-line%20quick%20start&color=blue&style=for-the-badge&cacheSeconds=3600)](https://docs.ipfs.io/how-to/command-line-quick-start/)
```

Comment:
The badge sizes are inconsistent. In general, I'd go for the smaller size.

Reply:
Okay I reduced it.